sktime/classification/early_classification/__init__.py

```diff
@@ -10,4 +10,4 @@
     ProbabilityThresholdEarlyClassifier,
 )
 from sktime.classification.early_classification._teaser import TEASER
-from sktime.classification.early_classification.base import BaseEarlyClassifier
+from sktime.classification.early_classification.old_base import BaseEarlyClassifier
```

Comment:
Can it stay base without renaming it to old_base? And just contain deprecation stuff?

Reply:
fair point